Windows 10 1903 Removed Features: updates and clarifications (#4352)
MicrosoftDocs/windows-itpro-docs: windows/deployment/planning/windows-10-1903-removed-features.md

> Applies to: Windows 10, version 1903

> Each version of Windows 10 adds new features. Occasionally, we also remove features, often because we've added a better option. Below are the details about the features that we've removed in Windows 10, version 1903. **The list below is subject to change and might not include every affected feature.** 

Reviewer: ```suggestion
New features are added in each version of Windows 10. Occasionally features are also removed, often because a better option has been added. Details about features that have been removed in Windows 10, version 1903 are listed below. **The list is subject to change and might not include every affected feature.** 
```

**Note**: Join the [Windows Insider program](https://insider.windows.com) to get early access to new Windows 10 builds and test these changes yourself.

> ## Features we removed or will remove soon

Reviewer: ```suggestion
## Features that have been removed or will be removed soon
```

> The following features are removed from the installed product image for Windows 10, version 1903, or are planned for removal in an upcoming release. Apps depending on these features won't function in this release.   

Reviewer: ```suggestion
The following features are removed from the installed product image for Windows 10, version 1903, or are planned for removal in an upcoming release. Apps that depend on these features won't function in this release.   
```

> The following features are removed from the installed product image for Windows 10, version 1903, or are planned for removal in an upcoming release. Apps that depend on these features won't function in this release.   

Reviewer 2: ```suggestion
The following features are removed from the installed product image for Windows 10, version 1903, or are planned for removal in an upcoming release. Apps depending on these features won't function in this release.   
```

|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> |         XDDM-based remote display driver          | Starting with this release, Remote Desktop Services uses an Indirect Display Driver (IDD)—based on Windows Display Driver Model (WDDM)—a single session remote desktop. The support for remote display drivers based on Windows 2000 Display Driver Model (XDDM) will be removed in a future release. Independent Software Vendors (ISVs) that use XDDM-based remote display driver should plan a migration to the WDDM driver model. For more information on implementing remote indirect display driver, ISVs can reach out to [rdsdev@microsoft.com](mailto:rdsdev@microsoft.com). |

Reviewer: ```suggestion
|         XDDM-based remote display driver          | Starting with this release, Remote Desktop Services uses an Indirect Display Driver (IDD) - based on Windows Display Driver Model (WDDM) - a single session remote desktop. The support for remote display drivers based on Windows 2000 Display Driver Model (XDDM) will be removed in a future release. Independent Software Vendors (ISVs) that use XDDM-based remote display driver should plan a migration to the WDDM driver model. For more information on implementing remote indirect display driver, ISVs can reach out to [rdsdev@microsoft.com](mailto:rdsdev@microsoft.com). |
```

Author: You've replaced em dashes with ASCII hypen. Why?

Please see the following parts of the Microsoft Style Guide

- [Top 10 tips for Microsoft style and voice § Don't be spacey](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#dont-be-spacey)
- [Em dashes](https://docs.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes)

| The Messaging app on desktop doesn't offer messages sync |                                                                                                                          The desktop version of the Messaging app has a sync feature that can be used to sync SMS text messages received from Windows Mobile and keep a copy of them on the desktop. The sync feature has been removed. Because of this change, you will only be able to access messages from the device that have received them.                                                                                                                          |

> ## Features we’re no longer developing

Reviewer: ```suggestion
## Features that are no longer being developed
```

> We're no longer developing these features, and may remove them in a future release. Some features have been replaced with other features , while others are now available from different sources. 

Reviewer: ```suggestion
These features are no longer being developed, and may be removed in a future release. Some features have been replaced, while others are now available from different sources. 
```

If you have feedback about the proposed replacement of any of these features, you can use the [Feedback Hub app](https://support.microsoft.com/help/4021566/windows-10-send-feedback-to-microsoft-with-feedback-hub-app). 

|Feature    |Details|
|-----------|---------------------|
| Taskbar settings roaming| Roaming of taskbar settings is no longer being developed and we plan to disable this capability in a future release|
|Wi-Fi WEP and TKIP|In this release a warning message will appear when connecting to Wi-Fi networks secured with WEP or TKIP, which are not as secure as those using WPA2 or WPA3. In a future release, any connection to a Wi-Fi network using these old ciphers will be disallowed. Wi-Fi routers should be updated to use AES ciphers, available with WPA2 or WPA3. |

> |Windows To Go|Windows To Go is no longer being developed. It does not support feature updates and therefore does not enable you to stay current. It also requires a specific type of USB flash driver that is no longer supported by many OEMs.|

Reviewer 2: ```suggestion
|Windows To Go|Windows To Go is no longer being developed. It does not support feature updates and therefore does not enable you to stay current. It also requires a specific type of USB flash drive no longer supported by many OEMs.|
```
